spread.yaml

```diff
@@ -5,6 +5,7 @@ environment:
   PROJECT_PATH: /rockcraft
   SNAPD_TESTING_TOOLS: $PROJECT_PATH/tools/external/tools
   PATH: /snap/bin:$PATH:$SNAPD_TESTING_TOOLS:$PROJECT_PATH/tools/spread
+  SPHINX_STARTER_PACK: $PROJECT_PATH/sphinx-starter-pack
```

Comment:
this propagates to all tests, do we really need it here?

Reply:
no we only need it for one test